boards/arm/frdm_k32l2b3/doc/index.rst

```diff
@@ -0,0 +1,230 @@
+.. _frdm_k32l2b3:
+
+NXP FRDM-K32L2B3
+##############
```

Comment:
Title underlines need to be at least as long as the title above it, so add a couple more ``#``s